### PR TITLE
Rename `release.sh` `master_version` function

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: #46﻿

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -122,10 +122,10 @@ function git_push() {
    git push "${repo_url}" ${git_args} )
 }
 
-# Return the master version of a release.
+# Return the major+minor version of a release.
 # For example, "v0.2.1" returns "0.2"
 # Parameters: $1 - release version label.
-function master_version() {
```
Comment (chizhg) [inline release.sh:128]: Do we know which projects are using this function? This will be a breaking change to them.

**Review by chizhg — COMMENTED** [review]
/lgtm
/approve

/hold
Please feel free to /unhold if you have a good answer to the comment.

File: release.sh (hunk 1 above)
Comment (upodroid) [reply release.sh:128]: I don't think anyone should be, this is vendored.

```
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  k8s-test-infra   master  158⬇  $   cd ../serving/
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  serving   gke-v22  $   grep -r master_version .
./vendor/knative.dev/hack/release.sh:function master_version() {
./vendor/knative.dev/hack/release.sh:    major_minor_version="$(master_version "${last_version}")"
./vendor/knative.dev/hack/release.sh:    RELEASE_BRANCH="release-$(master_version "${RELEASE_VERSION}")"
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  serving   gke-v22  $   cd ../eventing/
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  eventing   kubernetes-labels  1⚑  $   grep -r master_version .
./vendor/knative.dev/hack/release.sh:function master_version() {
./vendor/knative.dev/hack/release.sh:    major_minor_version="$(master_version "${last_version}")"
./vendor/knative.dev/hack/release.sh:    RELEASE_BRANCH="release-$(master_version "${RELEASE_VERSION}")"
```

Comment (chizhg) [reply release.sh:128]: Got it, so it's like an internal function. Thanks for checking!

**Pull request merged**